image/templates/helm/shared/templates/02-scanner-06-deployment.yaml.htpl

```diff
@@ -12,6 +12,7 @@ metadata:
     {{- include "srox.labels" (list . "deployment" "scanner") | nindent 4 }}
   annotations:
     {{- include "srox.annotations" (list . "deployment" "scanner") | nindent 4 }}
+    openshift.io/required-scc: {{ include "srox.globalResourceName" (list . "stackrox-scanner") }}
```

Comment:
I think this SCC is never created - we need to double-check which SCC is used by scanner in reality.